### DISCUSSION TITLE
Some UI details I noticed

### DISCUSSION THREAD

**Discussion by nema69** [TriliumNext/Trilium#4338] [Ideas]
Hey everyone, 
recently I took a closer look at the UI and noticed some things which _in my opinion_ could be improved. If anyone found more or has other feedback/opinions, please comment.

1.
![tab](https://github.com/zadam/trilium/assets/61890712/60bf9a28-8724-4ce4-8f0e-84c748be401c)
When the panel is hidden the leftmost tab is right next to the launcher bar, without any gap. This looks especially akward when the theme uses rounded corners for the tabs as the default theme does. When the panel is visible the gutter provides a gap. 

2.
![button right](https://github.com/zadam/trilium/assets/61890712/28be61b8-1f02-4836-818d-81c7762f7a15)
Buttons on the right hand side have a border when hovering,
![leftbutton](https://github.com/zadam/trilium/assets/61890712/e6bbee85-0d7d-4fe5-a746-7c7d5cda9a3a)
whereas the buttons above the note do not. 
![window_button](https://github.com/zadam/trilium/assets/61890712/776acd1d-013b-4d3e-a551-2d62bcf26232)
Window buttons again change their background color when hovering. A more consistent behavior would be better imo.

3.
The main menu button is considerably bigger than the other launcher icons and doesn't line up with the tabs. 
![menubutton](https://github.com/zadam/trilium/assets/61890712/30a1c5d2-0798-4905-a6c1-f3ae800fbc3c)

I'm curious if other people feel the same way and maybe noticed more details.

**Comment by zadam** [comment]
> When the panel is hidden the leftmost tab is right next to the launcher bar, without any gap.

Actually there was no gap even when expanded. What looked like a gap was the split size control. I've added 5px.

> A more consistent behavior would be better imo.

I agree, but at the moment I don't know to what this should be unified. Might be more work.

> The main menu button is considerably bigger than the other launcher icons and doesn't line up with the tabs.

It is bigger intentionally since it's the main menu. IMHO it's ok to not be lined up with tabs, the default state of the sidebar is expanded.
